frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -76,6 +80,13 @@ pods:
         goal: FINISHED
         cmd: ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/nodetool snapshot -p {{TASKCFG_ALL_JMX_PORT}} -t $CASSANDRA_SNAPSHOT_NAME -- $CASSANDRA_KEYSPACES
         resource-set: sidecar-resources
+      upload-aws:
+        goal: FINISHED
+        cmd: >
+                chmod a+x {{S3CLI_VERSION}} && eval "echo \"$(sed 's/\"/\\\"/g' < s3config.json)\"" > config.json
```

Comment:
what's the `sed` doing?

could it make sense to just write config.json from scratch with an `echo` here?